accel_code/run.py

```diff
@@ -15,6 +16,9 @@ def get_parser():
                     formatter_class=RawTextHelpFormatter
                     )
 
+    parser.add_argument('project_root_directory',
```

Comment:
I was thinking this argument would represent the `mount point` for `vosslabhpc`, not where the accelerometer files are located.